mne/epochs.py

```diff
@@ -1380,7 +1398,7 @@ def _get_data(self, out=True, picks=None, item=None, *, units=None,
                     data = data[:, picks]
                 if units is not None:
                     data *= ch_factors[:, np.newaxis]
-                return data
+                return data[..., start:stop]
```

Comment:
Slices are always views (even if they are effectively otherwise a no-op) so the OWNDATA will fail

Reply:
so should I do a `.copy` here? :thinking:

Reply:
You could special-case `tmin = tmax = None` and not slice in that case...

Reply:
Thanks, that did it:

```python
            if start != 0 or stop != self.times.size:
                data = data[..., start:stop]
            return data
```